dlt/normalize/items_normalizers.py

```diff
@@ -677,15 +687,16 @@ def __call__(self, extracted_items_file: str, root_table_name: str) -> List[TSch
             must_rewrite = pyarrow.should_normalize_arrow_schema(
```

Comment:
`remove_null_columns_from_schema` should be done inside `should_normalize_arrow_schema` and only one `must_rewrite` should be returned